package.json

```diff
@@ -9,6 +9,7 @@
         "help:dev:2": "echo Note the server will hot-reload the code from any",
         "help:dev:3": "echo source files as they change, to ease development.",
         "dev": "npm run typecheck && vite",
+        "dev:workbench": "npm run typecheck && vite --mode workbench",
```

Comment:
Needs its help messages documenting what it does.

Reply:
Done. While I was doing this, I noticed and fixed some bugs in other help scripts.